### ISSUE TITLE
implement method: Sensor::LocalSensorTime()

### ISSUE THREAD

**Issue by AstroKEW78** [USGS-Astrogeology/SensorUtils#57]
Local Solar Time - Sensor::LocalSensorTime

**Comment by jessemapel** [comment]
Is this suppose to be LocalSolarTime? Sensor doesn't appear to have a LocalSensorTime method.

**Comment by jessemapel** [comment]
I'm going to go ahead and implement this as LocalSolarTime

**Comment by AstroKEW78** [comment]
Yes, I think Stuart (or whoever wrote the original list) meant solar time.

**Comment by jessemapel** [comment]
This is blocked by #44 because it's used in the computations
